doc/content/verification_and_validation/verification_standards.md

```diff
@@ -0,0 +1,36 @@
+3. +Unit Clarity+: Utilize comments within the script to specify the units for all values employed in the analytical solution.
+4. +Standardized Constants+: Whenever possible, employ values from [PhysicalConstants](source/utils/PhysicalConstants.md).
+5. +Gold File Verification+: The Python scripts should run on the gold files associated with the verification case.
+6. +Visualization Consistency+: In verification plots, consistently use solid lines to represent TMAP8 results and dashed lines for analytical solutions. Points should only be used when the focus is on specific data points (e.g., validation or benchmarking).
+7. +Write Pythonic code+: When writing Python scripts, aim to follow the [PEP8](https://peps.python.org/pep-0008/) style guide. A key principle is to use existing Python functionality whenever possible instead of writing your own code to do the same thing.
+
+## Tests
+
+1. +Comprehensive Testing+: Every input file for verification should have corresponding [EXODIFF](https://mooseframework.inl.gov/moose/python/testers/Exodiff.html) and [CSVDIFF](https://mooseframework.inl.gov/moose/python/testers/CSVDiff_tester.html) tests to ensure accuracy. EXODIFF tests are not required if the simulation is 0D (single element), and the [PostProcessors](https://mooseframework.inl.gov/moose/syntax/Postprocessors/index.html) to the CSVDIFF already test all the important variables.
+2. +Test specification+: Test requirements should detail what the particular test is checking, as well as mention the general physics the tested input file is modeling.
+3. +Heavy tests+: If a heavy test is used for verification, the test specification should mention that a finer mesh and/or small time step size is being used for verification. A test can be declared to be a heavy test by adding the ```heavy = true``` to the test specification.
+4. +Script Verification+: The Python scripts used for verification should themselves be subjected to testing to guarantee their reliability.
+
+## Documentation
+
+1. +Automated Figure Generation+: Figures used in the verification process should be automatically generated during the build phase by executing the dedicated plotting Python script.
+2. +Analytical Equations+: Ensure the equations documented exactly match those used within the Python script. Always cite the original sources for the equations (not a verification report) to maintain proper attribution.
+3. +Detailed Derivations+: If any quantity conversions are performed within the documentation, provide clear derivations to illustrate the conversion process.Show any derivations that are performed for converting quantities in the documentation.
+4. +Schematic Representation+: Include schematic figures that visually represent what the specific verification case is modeling to enhance understanding.
```

Comment:
```suggestion
5. +Schematic Representation+: Whenever helpful, include schematic figures that visually represent what the specific verification case is modeling to enhance understanding.
```
I do not think that an illustration is useful in several cases, so I am nuancing this guideline a little bit.